index.js

```diff
@@ -11,26 +13,26 @@ const debug = require('debug')('detective-sass');
  * @return {String[]}
  */
 module.exports = function detective(fileContent) {
-  if (typeof fileContent === 'undefined') { throw new Error('content not given'); }
-  if (typeof fileContent !== 'string') { throw new Error('content is not a string'); }
+  if (typeof fileContent === 'undefined') throw new Error('content not given');
+  if (typeof fileContent !== 'string') throw new Error('content is not a string');
+  if (fileContent === '') return [];
```

Comment:
@mrjoelkemp this is how I worked around the strict error here and in the scss repo. How do you think we should proceed? We could land this as is, or make it a major version bump and throw an error.